### PR TITLE
Add UART version PMW3901 driver enabling method (SENS_TFLOW_CFG)

### PR DESCRIPTION
**Pull request by bys1123**

### CODE REVIEW

File: en/sensor/pmw3901.md [hunk 1]
```diff
@@ -71,7 +71,10 @@ The PX4 configuration that is common to all PMW3901-based boards:
 
 In addition:
 - The distance sensors should be enabled as described in their setup documentation (no flow-specific configuration is required).
-  For sensors connected by SPI or I2C (but not UAVCAN), [SENS_EN_PMW3901](../advanced_config/parameter_reference.md#SENS_EN_PMW3901) must be set `1` in order to enable the sensor driver.
+  For sensors connected by SPI or I2C (but not UAVCAN/UART), [SENS_EN_PMW3901](../advanced_config/parameter_reference.md#SENS_EN_PMW3901) must be set `1` in order to enable the sensor driver.
```
Comment (hamishwillee) [inline en/sensor/pmw3901.md:74]: Thanks @bys1123 !

I've done a minor subedit and I plan to merge shortly (at some point in future might make sense for us to split the docs for each sensor up. 

Anyway, one question, for the UART sensor do I STILL need to set `SENS_EN_PMW3901`? Or is setting the UART enough?
My guess is that it is, but wanted make sure.

Comment (bys1123) [reply en/sensor/pmw3901.md:74]: SENS_EN_PMW3901 only enabled SPI version PMW3901 driver, (I don't think this driver have I2C supported, needs to confirm). So, no need to enable it for UART version.

File: en/sensor/pmw3901.md [hunk 2]
```diff
@@ -182,3 +185,6 @@ The board doesn't include a distance sensor onboard so you will need to use an [
 <!-- note, this will be set using SENS_TFLOW_CFG
 Wiring also required.
 -->
+In addition to the general [PX4 configuration](#px4-configuration) you must also set the parameter [SENS_TFLOW_CFG](../advanced_config/parameter_reference.md#SENS_TFLOW_CFG) to the value of the UART port you connected (e.g. if the sensor is connected to `TELEM 2` then set `SENS_TFLOW_CFG=102`.
+For more information see Serial Port Configuration](../peripherals/serial_configuration.md).
```
Comment (bys1123) [inline en/sensor/pmw3901.md:189]: [Serial Port Configuration]
missing `[`

Comment (mrpollo) [reply en/sensor/pmw3901.md]: @bys1123 I fixed this for you.

**Comment by mrpollo** [comment]
@hamishwillee @bys1123 this PR is ready to go

File: en/sensor/pmw3901.md [hunk 3]
```diff
@@ -30,7 +30,7 @@ An external rangefinder/distance sensor is *required* for the sensors that don't
 
 The range needed is dependent on the application:
 - Indoor flight: ≈4 meters
-- Outdoor flight: ≥10 metres (e.g. to support the position control in environments where GPS might have issues)
+- Outdoor flight: ≥10 meters (e.g. to support the position control in environments where GPS might have issues)
```
Comment (hamishwillee) [inline en/sensor/pmw3901.md:33]: FYI, metres is correct spelling for UK English, the language of this doc set. Pretty much only USA and Philippines use "meter".

File: en/sensor/pmw3901.md [hunk 4]
```diff
@@ -42,15 +42,15 @@ The PX4 team mainly tested using the [Lidar Lite V3](../sensor/lidar_lite.md) on
 <span id="orientation"></span>
 ## Mounting/Orientation
 
-Flow modules are typically mounted near the centre of the vehicle.
+Flow modules are typically mounted near the center of the vehicle.
```
Comment (hamishwillee) [inline en/sensor/pmw3901.md:45]: Ditto "center" is US spelling.

File: en/sensor/pmw3901.md [hunk 5]
```diff
@@ -42,15 +42,15 @@ The PX4 team mainly tested using the [Lidar Lite V3](../sensor/lidar_lite.md) on
...
 If mounted off-centre you will need to set offsets: [Optical Flow > EKF2](../sensor/optical_flow.md#ekf2).
 
-The flow modules may be mounted with any yaw orientation relative to vehicle body frame, but you must set the value used in [SENS_FLOW_ROT](../advanced_config/parameter_reference.md#SENS_FLOW_ROT).
+The flow modules may be mounted with any yaw orientation relative to the vehicle body frame, but you must set the value used in [SENS_FLOW_ROT](../advanced_config/parameter_reference.md#SENS_FLOW_ROT).
```
Comment (hamishwillee) [inline en/sensor/pmw3901.md:48]: Previous text was correct and this is incorrect. It must be either "to vehicle body frame" or "to the vehicle's body frame". The former is more concise.

File: en/sensor/pmw3901.md [hunk 6]
```diff
@@ -42,15 +42,15 @@ The PX4 team mainly tested using the [Lidar Lite V3](../sensor/lidar_lite.md) on
...
 
 The "zero" rotation is when the sensor board and [vehicle](../getting_started/px4_basic_concepts.md#heading-and-directions) X-axes are aligned (i.e. the "front" of the vehicle and the board are in the same direction), with rotations increasing in a clockwise direction.
 
 PMW3901 modules have a small notch indicating the direction of the **back** of the board. 
-The diagram shows the orientation of the Bitcraze board to the vehicle that corresponds to `SENS_FLOW_ROT=0` (note the notch at the back).
+The diagram shows the Bitcraze board's orientation to the vehicle that corresponds to `SENS_FLOW_ROT=0` (note the notch at the back).
```
Comment (hamishwillee) [inline en/sensor/pmw3901.md:53]: This is worse grammar.

File: en/sensor/pmw3901.md (hunk 1 above)
Comment (hamishwillee) [reply en/sensor/pmw3901.md:74]: Thanks, I'll update.

File: en/sensor/pmw3901.md [hunk 7]
```diff
@@ -107,8 +109,8 @@ Pixhawk SPI Port (from left to right) | Bitcraze flow board
 6 (CS2) | CS
 7 (GND) | GND
 
-In order to connect the bitcraze flow board to the Pixhawk you will need to solder the wires of the Pixhawk SPI cable to the flow board.
-An SPI cable has 7 wires, from which you need to connect 6 to the flow board.
+To connect the bitcraze flow board to the Pixhawk, you will need to solder the wires of the Pixhawk SPI cable to the flow board.
+An SPI cable has seven wires, from which you need to connect 6 to the flow board.
```
Comment (hamishwillee) [inline en/sensor/pmw3901.md:113]: Why change 7 to seven and not 6 to six? Personally I prefer the use of numbers EVEN though using the word is technically more correct. It is more concise and easier to read.

File: en/sensor/pmw3901.md [hunk 8]
```diff
@@ -139,23 +141,22 @@ The AlienTek [ATK-PMW3901](https://www.aliexpress.com/i/32979605707.html) expose
 
 ![Alientek Pixhawk4 Connections](../../assets/hardware/sensors/pmw3901/ph4-alientek-flow-pinout.png)
 
-The board also incorporates a distance sensor (again, we recommend that you use an [external distance sensor](#external_rangefinder)).
-You can wire the internal sensor to the Pixhawk I2C port [in the same way as any other I2C peripheral](#i2c_wiring) 
-A screenshot showing the I2C pins (SLA, SLC, GND and VCC) is provided below.
+The board also incorporates a distance sensor (We recommend you use an [external distance sensor](#external_rangefinder)).
```
Comment (hamishwillee) [inline en/sensor/pmw3901.md:144]: We should be lower case as it is part of the sentence, also missing "that). Removal of the "again" is good. That kind of extra word is unnecessary.

File: en/sensor/pmw3901.md [hunk 9]
```diff
@@ -139,23 +141,22 @@ The AlienTek [ATK-PMW3901](https://www.aliexpress.com/i/32979605707.html) expose
...
+A screenshot showing the I2C pins (SLA, SLC, GND, and VCC) is provided below.
 
 ![Alientek Pinout](../../assets/hardware/sensors/pmw3901/alientek.png)
 
-
 <span id="hex_hereflow_pwm3901"></span>
 ## Hex HereFlow PMW3901 Optical Flow Sensor
 
-The Hex [HereFlow PMW3901 Optical Flow Sensor](http://www.proficnc.com/all-products/185-pixhawk2-suite.html) is a very small board containing the PMW3901 flow module, the VL53L1X distance sensor and an IMU to better synchronize the flow data with the gyro data.
+The Hex [HereFlow PMW3901 Optical Flow Sensor](http://www.proficnc.com/all-products/185-pixhawk2-suite.html) is a tiny board containing the PMW3901 flow module, VL53L1X distance sensor, and an IMU to synchronize the flow gyro data better.
```
Comment (hamishwillee) [inline en/sensor/pmw3901.md:153]: > and an IMU to synchronize the flow gyro data better.

This has changed the meaning, and I *think* is now incorrect

**Comment by hamishwillee** [comment]
@mrpollo Thanks. I'm going to revert some of the grammatical and spelling changes. 
- Currently the doc language is UK English, so metre not meter
- I think "7" is more readable than "seven" - while not as correct if you were writing an essay, very common and more readable in technical docs.

Those kinds of things are arguable, but should be done globally, not in just one doc. 

I also need to tidy up the sensor settings info. The sensors are a bit irritating in that each bus has a different parameter that needs to be set!

File: en/sensor/pmw3901.md (hunk 5 above)
Comment (hamishwillee) [reply en/sensor/pmw3901.md:48]: Actually, I am not sure on this. Leaving as is.

**Comment by hamishwillee** [comment]
Updated and merging. Thanks @bys1123 and @mrpollo for taking the time to make sure this is right!

**Comment by bys1123** [comment]
Thanks! @hamishwillee

**Pull request merged**